jetty-osgi/test-jetty-osgi/src/test/java/org/eclipse/jetty/osgi/test/TestJettyOSGiBootHTTP2JDK9.java

```diff
@@ -0,0 +1,164 @@
+        res.add(mavenBundle().groupId("org.eclipse.jetty").artifactId("jetty-alpn-java-server").versionAsInProject().start());
+        res.add(mavenBundle().groupId("org.eclipse.jetty").artifactId("jetty-alpn-server").versionAsInProject().start());
+
+        res.add(mavenBundle().groupId("org.eclipse.jetty.http2").artifactId("http2-common").versionAsInProject().start());
+        res.add(mavenBundle().groupId("org.eclipse.jetty.http2").artifactId("http2-hpack").versionAsInProject().start());
+        res.add(mavenBundle().groupId("org.eclipse.jetty.http2").artifactId("http2-server").versionAsInProject().start());
+        return res;
+    }
+ 
+
+    @Ignore
+    public void assertAllBundlesActiveOrResolved() throws Exception
+    {
+        TestOSGiUtil.debugBundles(bundleContext);
+        TestOSGiUtil.assertAllBundlesActiveOrResolved(bundleContext);
+        Bundle javaAlpn = TestOSGiUtil.getBundle(bundleContext, "org.eclipse.jetty.alpn.java.server");
+        assertNotNull(javaAlpn);
+        ServiceReference<?>[] services = javaAlpn.getRegisteredServices();
+        assertNotNull(services);        
+        Bundle server = TestOSGiUtil.getBundle(bundleContext, "org.eclipse.jetty.alpn.server");
+        assertNotNull(server);
+    }
+
+    
+
+    @Test
+    public void testHTTP2() throws Exception
+    {
+        HttpClient httpClient = null;
+        HTTP2Client http2Client = null;
+        try 
+        {
+            //get the port chosen for https
+            String port = System.getProperty("boot.https.port");
+            assertNotNull(port);
+            
+            Path path = Paths.get("src",  "test", "config");
+            File keys = path.resolve("etc").resolve("keystore").toFile();
+            
+            //set up client to do http2
+            http2Client = new HTTP2Client();
+            SslContextFactory sslContextFactory = new SslContextFactory();
+            sslContextFactory.setKeyManagerPassword("OBF:1vny1zlo1x8e1vnw1vn61x8g1zlu1vn4");
+            sslContextFactory.setTrustStorePath(keys.getAbsolutePath());
+            sslContextFactory.setKeyStorePath(keys.getAbsolutePath());
+            sslContextFactory.setTrustStorePassword("OBF:1vny1zlo1x8e1vnw1vn61x8g1zlu1vn4");
+            httpClient = new HttpClient(new HttpClientTransportOverHTTP2(http2Client), sslContextFactory);
+            Executor executor = new QueuedThreadPool();
+            httpClient.setExecutor(executor);
+            httpClient.start();
+
+            ContentResponse response = httpClient.GET("https://localhost:"+port+"/jsp/jstl.jsp");
+            assertEquals(200, response.getStatus());
+            assertTrue(response.getContentAsString().contains("JSTL Example"));
+
+        }
+        finally
+        {
+            if (httpClient != null) httpClient.stop();
+            if (http2Client != null) http2Client.stop();
+        }
+    }
+
+}
```

Comment:
If you can remove all the unneeded empty lines between methods, at the end of methods, etc. - looks tidier without them.